OpenRA.Game/Settings.cs

```diff
@@ -90,18 +90,36 @@ public class DebugSettings
 		public bool SanityCheckUnsyncedCode = false;
 		public int Samples = 25;
 		public bool IgnoreVersionMismatch = false;
+
+		[Desc("Dump performance data into cpu.csv and render.csv accessible via ~/.openra/Logs.")]
```

Comment:
I'm with @RoosterDragon in that you could say `via your content log directory` or something to that effect.